ipet/parsing/StatisticReader.py

```diff
@@ -193,23 +193,6 @@ def extractStatistic(self, line):
                 self.addData(key, timestamp)
                 break
 
-class PathReader(StatisticReader):
```

Comment:
Note that "@01" is a modifiable argument in the reader manager, and we parse it now in the reader manager. This way, we are both faster, use less code, and are more flexible.